app/assets/javascripts/admin/app/controllers/person_controller.js

```diff
@@ -56,7 +56,7 @@ this.GobiertoAdmin.PersonController = (function() {
       items: {
         src: '#crop-popup'
       },
-      // mainClass: 'modal-full-width',
+      mainClass: 'modal-full-width',
```

Comment:
why recover it?

Reply:
Without this class, with medium images 600x900 or 900x600, the popup adapts to the image size but with the class 'modal-full-width' large images, make the popup giant. Any solution?